proto/centauri/ratelimit/v1beta1/params.proto

```diff
@@ -7,4 +7,9 @@ import "gogoproto/gogo.proto";
 import "cosmos_proto/cosmos.proto";
 
 // Params holds parameters for the mint module.
-message Params {}
+message Params {
```

Comment:
this needs to be per token, right?

Reply:
Yeah, Thanks for your review. I'll update now